articles/tutorials/building_2d_games/04_working_with_textures/index.md

```diff
@@ -0,0 +1,344 @@
+---
+title: "Chapter 04: Working with Textures"
+description: In this chapter, we'll look at the process of loading image files as textures using the MonoGame content pipeline and rendering them using the SpriteBatch.
+---
+
+Textures are images that are used in your game to represent the visual graphics to the player.  Before being able to render them with the [SpriteBatch](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch), you first need to create or load the texture.  We'll discuss creating a texture in code later in this series. For now, we're going to look at loading existing image files as textures and then rendering them with the [SpriteBatch](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch).
+
+## Loading a Texture
+
+Loading textures, like any other asset in MonoGame, can be done by loading the asset file directly at runtime in game, or it can be loaded through the *content pipeline*.  Both of these methods are two sides of the same coin and there are trade offs to each approach.
+
+For instance, to load an image file directly at runtime, you would need to
+
+1. Add the image file to your project
+2. Configure the project to copy the image file to the game project build directory
+3. Load the image file as a texture at runtime using the [Texture2D.FromFile](xref:Microsoft.Xna.Framework.Graphics.Texture2D.FromFile(Microsoft.Xna.Framework.Graphics.GraphicsDevice,System.String)) method.
+
+When loading an image file as a texture directly like this, however, loads it in its compressed format such as *.png* or *.jpg*.  These compression formats are not understood by a Graphics Processing Unit (GPU); they will need to be decompressed into raw bytes as a format the GPU does understand before it can store the data.  Doing this can potentially leave a larger memory footprint for your assets.

```

Comment:
Make this an "Important" section as it REALLY needs highlighting about the disadvantages of just "loading it yourself".  Maybe also add one of Tom's points from the last AMA, where different formats may be needed for different platforms, where as the Content Pipeline just "handles it for you" :D

Reply:
This has been moved out into an Important admonition section with language to indicate that developers would need to handle this per platform and that the content pipeline handles it automatically.